### PR TITLE
feat: Added Docs for Slash CLI

### PR DESCRIPTION
**Pull request by antblood**
Added Docs for Slash CLI.

### CODE REVIEW

**Comment by bucanero** [comment]
hello @antblood 👋 

I think that we could group the Slash CLI articles in a simplified way, to help the user navigate the content. 
My idea would be to group content in:

- Slash CLI
  - Backend (deploy, destroy, drop, export, list, restore, update)
  - Lambda (delete, get, logs, update)
  - Schema (get, update)
  - login & logout
  - Update CLI & help

what do you think?

File: content/slash-cli/deploy-backend.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
++++
+title = "Deploy Backend"
+weight = 2
+[menu.main]
+    parent = "slash-graphql-cli"
+    identifier = "deploy-backend"
++++
+
+Launch a new Backend.
+
```
Comment (aaroncarey) [inline content/slash-cli/deploy-backend.md:10]: Backend. -> backend.

File: content/slash-cli/delete-lambda.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+Delete the Lambda script associated with the backend.
+
+USAGE
+```
+  $ slash-graphql delete-lambda
+```
+OPTIONS
+```
+  -e, --endpoint=endpoint  Slash GraphQL Endpoint
```
Comment (aaroncarey) [inline content/slash-cli/delete-lambda.md:17]: Outside of titles and headings, I'd suggest only having Slash GraphQL, Dgraph, or the first word in a sentence capitalized. Please apply this change to all of these files, time permitting (i.e., Slash GraphQL Endpint -> Slash GraphQL endpoint). Thanks :)

File: content/slash-cli/drop.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
++++
+title = "Drop"
```
Comment (aaroncarey) [inline content/slash-cli/drop.md:2]: I'd suggest "Drop data".

Also, if "drop data" has a meaning distinct from "delete data", please help me with a definition for our terminology and glossary efforts :)

File: content/slash-cli/destroy-backend.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
++++
+title = "Destroy Backend"
```
Comment (aaroncarey) [inline content/slash-cli/destroy-backend.md:2]: Does "destroy backend" have a meaning distinct from "delete backend"? If so, please help me with a definition for our terminology and glossary efforts, which we might also use to enhance the description on this page. 

If not, I'd suggest changing the description below to "Destroy (i.e., delete) a backend by id."

File: content/slash-cli/drop.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+weight = 2
+[menu.main]
+    parent = "slash-graphql-cli"
+    identifier = "drop"
++++
+
+Drop all data in your backend.
```
Comment (aaroncarey) [inline content/slash-cli/drop.md:9]: If "drop" is a synonym for "delete", please update the description to:

"Drop (i.e., delete) all data in your backend."

File: content/slash-cli/get-lambda.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
++++
+title = "Get Lambda"
```
Comment (aaroncarey) [inline content/slash-cli/get-lambda.md:2]: I'd suggest "Get Lambda Script" for the title

File: content/slash-cli/import-data.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
++++
+title = "Import Data"
+weight = 2
+[menu.main]
+    parent = "slash-graphql-cli"
+    identifier = "import-data"
++++
+
+Import your data back via live loader (requires docker).
```
Comment (aaroncarey) [inline content/slash-cli/import-data.md:9]: "via live loader" -> "using Dgraph Live Loader" (per usage example here: https://dgraph.io/docs//deploy/fast-data-loading/#live-loader)

File: content/slash-cli/lambda-logs.md [hunk 8]
```diff
@@ -0,0 +1,28 @@
++++
+title = "Lambda Logs"
+weight = 2
+[menu.main]
+    parent = "slash-graphql-cli"
+    identifier = "lambda-logs"
++++
+
+Get the Lambda script associated with the backend.
```
Comment (aaroncarey) [inline content/slash-cli/lambda-logs.md:9]: script -> logs

File: content/slash-cli/list-backups.md [hunk 9]
```diff
@@ -0,0 +1,26 @@
++++
+title = "List Backups"
+weight = 2
+[menu.main]
+    parent = "slash-graphql-cli"
+    identifier = "list-backups"
++++
+
+List all backups of the current backend.
```
Comment (aaroncarey) [inline content/slash-cli/list-backups.md:9]: current -> currently selected

File: content/slash-cli/overview.md [hunk 10]
```diff
@@ -0,0 +1,71 @@
...
+
+* [Get Lambda](/slash-cli/get-lambda) Get the Lambda script associated with the backend.
+
+* [Get Schema](/slash-cli/get-schema) Fetch the schema from your backend.
+
+* [Help](/slash-cli/help) Display help for slash-graphql.
+
+* [Import Data](/slash-cli/import-data) Import your data back via live loader (requires docker).
+
+* [Lambda Logs](/slash-cli/lambda-logs) Get the Lambda script associated with the backend.
```
Comment (aaroncarey) [inline content/slash-cli/overview.md:52]: script -> logs

File: content/slash-cli/overview.md [hunk 11]
```diff
@@ -0,0 +1,71 @@
...
+
+* [Drop](/slash-cli/drop) Drop all data in your backend.
+
+* [Export Data](/slash-cli/export-data) Export data from your backend.
+
+* [Get Lambda](/slash-cli/get-lambda) Get the Lambda script associated with the backend.
+
+* [Get Schema](/slash-cli/get-schema) Fetch the schema from your backend.
+
+* [Help](/slash-cli/help) Display help for slash-graphql.
```
Comment (aaroncarey) [inline content/slash-cli/overview.md:48]: Display help for the `slash-graphql` command line.

File: content/slash-cli/overview.md [hunk 12]
```diff
@@ -0,0 +1,71 @@
...
+title = "Overview"
+weight = 1
+[menu.main]
+    parent = "slash-graphql-cli"
+    identifier = "Overview"
++++
+
+*These are draft docs for Slash GraphQL, which is currently in beta*
+
+Manage Slash GraphQL from the comfort of your command line!
```
Comment (aaroncarey) [inline content/slash-cli/overview.md:12]: I'd suggest:

Slash GraphQL now includes a CLI so you can manage it from the comfort of your command line!

File: content/slash-cli/help.md [hunk 13]
```diff
@@ -0,0 +1,24 @@
++++
+title = "Help"
+weight = 2
+[menu.main]
+    parent = "slash-graphql-cli"
+    identifier = "help"
++++
+
+Display help for slash-graphql.
```
Comment (aaroncarey) [inline content/slash-cli/help.md:9]: I'd suggest the following decscription:

Display help for the `slash-graphql` CLI.

File: content/slash-cli/overview.md [hunk 14]
```diff
@@ -0,0 +1,71 @@
...
+
+* [Export Data](/slash-cli/export-data) Export data from your backend.
+
+* [Get Lambda](/slash-cli/get-lambda) Get the Lambda script associated with the backend.
+
+* [Get Schema](/slash-cli/get-schema) Fetch the schema from your backend.
+
+* [Help](/slash-cli/help) Display help for slash-graphql.
+
+* [Import Data](/slash-cli/import-data) Import your data back via live loader (requires docker).
```
Comment (aaroncarey) [inline content/slash-cli/overview.md:50]: via live loader -> using Dgraph Live Loader

File: content/slash-cli/overview.md [hunk 15]
```diff
@@ -0,0 +1,71 @@
...
+
+* [Lambda Logs](/slash-cli/lambda-logs) Get the Lambda script associated with the backend.
+
+* [List Backends](/slash-cli/list-backends) List your backends.
+
+* [Login](/slash-cli/login) Login to Slash GraphQL. Calling this function will keep you logged in for 24 hours, and you will not need to pass access tokens for any backends that you own.
+
+* [Logout](/slash-cli/logout) Logout of Slash GraphQL Command Line.
```
Comment (aaroncarey) [inline content/slash-cli/overview.md:58]: Command Line -> CLI

File: content/slash-cli/overview.md [hunk 16]
```diff
@@ -0,0 +1,71 @@
...
+
+* [Restore Backend](/slash-cli/restore-backend) Restore into a backend by source backend ID.
+
+* [Restore Backend Status](/slash-cli/restore-backend-status) Retrieve the status of a restore operation.
+
+* [Update](/slash-cli/update) Update the slash-graphql CLI.
```
Comment (aaroncarey) [inline content/slash-cli/overview.md:64]: slash-graphql -> `slash-graphql`

File: content/slash-cli/update.md [hunk 17]
```diff
@@ -0,0 +1,14 @@
++++
+title = "Update"
+weight = 2
+[menu.main]
+    parent = "slash-graphql-cli"
+    identifier = "update"
++++
+
+Update the slash-graphql CLI.
```
Comment (aaroncarey) [inline content/slash-cli/update.md:9]: Update the `slash-graphql` CLI.

**Review by aaroncarey — APPROVED** [review]
LGTM after a few small changes. Thanks for the chance to review this :)

**Comment by antblood** [comment]
Thanks a lot for the wonderful reviews. I made the necessary changes, can you please have another look at this PR?

**Review by aaroncarey — APPROVED** [review]

**Comment by aaroncarey** [comment]
> hello @antblood 👋
> 
> I think that we could group the Slash CLI articles in a simplified way, to help the user navigate the content.
> My idea would be to group content in:
> 
>     * Slash CLI
>       
>       * Backend (deploy, destroy, drop, export, list, restore, update)
>       * Lambda (delete, get, logs, update)
>       * Schema (get, update)
>       * login & logout
>       * Update CLI & help
> 
> 
> what do you think?
👍

File: content/slash-cli/login-logout.md [hunk 18]
```diff
@@ -0,0 +1,50 @@
++++
+title = "Login & Logout"
+weight = 2
+[menu.main]
+    parent = "slash-graphql-cli"
+    identifier = "login-logout"
```
Comment (bucanero) [inline content/slash-cli/login-logout.md:6]:
```suggestion
title = "Login"
weight = 2
[menu.main]
    parent = "slash-graphql-cli"
    identifier = "login"
```

File: content/introduction.md [hunk 19]
```diff
@@ -16,6 +16,14 @@ Please see the following topics:
   - [Import and Exporting Data](/admin/import-export) is a guide for exporting your data from a Slash GraphQL backend, and how to import it into another cluster
   - [Dropping Data](/admin/drop-data) will guide you through dropping all data from your Slash GraphQL backend.
   - [Switching Backend Modes](/admin/backend-modes) will guide you through changing Slash GraphQL backend mode.
+- [Slash GraphQL CLI](/slash/overview) Slash GraphQL now includes a CLI so you can manage it from the comfort of your command line
+  - [Backend](/slash-cli/backend) lists commands related to backend.
+  - [Backup](/slash-cli/backup) lists commands related to backup.
+  - [Help](/slash-cli/help-update) lists commands to update `slash-graphql` CLI and help.
+  - [Lambda](/slash-cli/lambda) lists commands related to Lambda.
+  - [Login & Logout](/slash-cli/login-logout) lists login and logout commands.
```
Comment (bucanero) [inline content/introduction.md:24]:
```suggestion
  - [Login](/slash-cli/login) lists login and logout commands.
```

**Review by bucanero — APPROVED** [review]
hello @antblood , everything looks great! 👍 

btw, if you like, the `Login & Logout` title could be shortened to `Login`

also, since the Overview is short, if you want, the `Help & Update` could be moved (merged) to the Overview page.

**Comment by bucanero** [comment]
@antblood, Everything looks great, feel free to merge the changes 👍

**Comment by mbj36** [comment]
Woohoo!

**Pull request merged**